app/src/pages/inside/settingsPage/generalTab/generalTab.jsx

```diff
@@ -0,0 +1,321 @@
+                value={this.state.inactivityTimeout}
+                options={optionsLaunch}
+              />
+            </div>
+            <div className={cellClasses}>
+              <span>Schedule time for Job to interrupt inactive launches</span>
+            </div>
+          </div>
+
+          <div className={styles.formField}>
+            <div className={cellClasses}>
+              <label htmlFor={'keepLogs'} className={styles.generalTabLabel}>
+                Keep logs
+              </label>
+            </div>
+            <div className={cellClassesFormField}>
+              <InputDropdown
+                id={'keepLogs'}
+                onChange={this.onLogsChange}
+                value={this.state.keepLogs}
+                options={options2}
+              />
+            </div>
+            <div className={cellClasses}>
+              <span>
+                How long to keep old logs in launches. Related launches structure will be saved, in
+                order to keep statistics
+              </span>
+            </div>
+          </div>
+
+          <div className={styles.formField}>
+            <div className={cellClasses}>
+              <label htmlFor={'keepAttachments'} className={styles.generalTabLabel}>
+                Keep attachments
+              </label>
+            </div>
+            <div className={cellClassesFormField}>
+              <InputDropdown
+                id={'keepAttachments'}
+                onChange={this.onAttachmentsChange}
+                value={this.state.keepAttachments}
+                options={options2}
+              />
+            </div>
+            <div className={cellClasses}>
+              <span>How long to keep attachments in system</span>
+            </div>
+          </div>
+
+          <div className={styles.formField}>
+            <div className={cellClasses}>
+              <label htmlFor={'autoAnalysis'}>Auto analysis</label>
+            </div>
+            <div className={cellClasses}>
+              <InputBigSwitcher
+                id={'autoAnalysis'}
+                onChange={this.onAnalysisChange}
+                value={this.state.autoAnalysis}
+                leftAligned
+              />
+            </div>
+            <div className={cellClassesColumn}>
+              <span>If ON - analysis starts as soon as any launch finished.</span>
+              <span>If OFF - not automatic, but can be invoked manually.</span>
+            </div>
+          </div>
+
+          {this.state.autoAnalysis && (
+            <div>
+              <div className={styles.formField}>
+                <div className={cellClasses}>
+                  <div>Base for Auto Analysis</div>
+                </div>
+
+                <div className={cellClasses}>
+                  <label className={cx('radioCheckbox')}>
+                    <div className={styles.CircleOuter}>
+                      {this.state.autoAnalysisBase === 'allLaunches' && (
```

Comment:
Comparing with a 'magic string' can be risky. Any typo or a similar character in another language can be a problem that cannot be identified easily. I suggest using constants in all such places.